Add a warning about third-party cookies blocking in Google Chrome (#32612)
mdn/content: files/en-us/web/http/headers/set-cookie/index.md

        > **Note:** A [`Secure`](#secure) cookie is only sent to the server with an encrypted request over the HTTPS protocol. Note that insecure sites (`http:`) can't set cookies with the `Secure` directive, and therefore can't use `SameSite=None`.

>         > **Warning:** Cookies with the `SameSite=None; Secure` that do not also have the [`Partitioned`](#partitioned) attribute may be blocked in cross-site contexts on future browser versions. This behavior protects user data from cross-site tracking. You may use this with [Storage Access API](/en-US/docs/Web/API/Storage_Access_API) or [Partitioned](#partitioned) attribute.

Reviewer: ok, revisiting. The bit "You may use this with [Storage Access API](/en-US/docs/Web/API/Storage_Access_API) or [Partitioned](#partitioned) attribute." doesn't make much sense with the rest of the sentence. Updating to remove that and cross link to more information.

```suggestion
        > **Warning:** Cookies with the `SameSite=None; Secure` that do not also have the [`Partitioned`](#partitioned) attribute may be blocked in cross-site contexts on future browser versions. This behavior protects user data from cross-site tracking. See [Cookies Having Independent Partitioned State (CHIPS)](/en-US/docs/Web/Privacy/Privacy_sandbox/Partitioned_cookies) and [Third-party cookies](https://developer.mozilla.org/en-US/docs/Web/Privacy/Third-party_cookies).
```

@wbamberg Is this sufficient? I'm not opposed to more work being done here, but I don't think I'd inject it to into this PR.

Reviewer 2: I think that's fine (for this PR anyway).

>         > **Warning:** Cookies with the `SameSite=None; Secure` and not `Partitioned` attributes that operate in cross-site contexts are third-party cookies. In future Google Chrome versions, reading third-party cookies will be blocked. This behavior protects user data from cross-site tracking. You may use this with [Storage Access API](/en-US/docs/Web/API/Storage_Access_API) or [Partitioned](#partitioned) attribute.

Reviewer: @xuanzhi33 Thanks very much. I think this is useful to capture so that people start using `Partitioned` where needed.

1. A third party cookies is one that is accessed in another origin - being `partitioned` makes no difference to that. So what I THINK you are saying here is that in future cross-site third party cookies will only be allowed for `SameSite=None; Secure` IFF they are also Partitioned - is that right?
2. Generally we shouldn't mention a particular browser here because it is "compatibility information". What we should do is make it clear this is likely to be a compatibility change, and when it happens we would record that in the compatibility data.

Does this capture the intent?

```suggestion
        > **Warning:** Cookies with the `SameSite=None; Secure` that do not also have the [`Partitioned`](#partitioned) attribute may be blocked in cross-site contexts on future browser versions. This behavior protects user data from cross-site tracking. You may use this with [Storage Access API](/en-US/docs/Web/API/Storage_Access_API) or [Partitioned](#partitioned) attribute.
```

Author: > @xuanzhi33 Thanks very much. I think this is useful to capture so that people start using `Partitioned` where needed.
> 
> 1. A third party cookies is one that is accessed in another origin - being `partitioned` makes no difference to that. So what I THINK you are saying here is that in future cross-site third party cookies will only be allowed for `SameSite=None; Secure` IFF they are also Partitioned - is that right?
> 2. Generally we shouldn't mention a particular browser here because it is "compatibility information". What we should do is make it clear this is likely to be a compatibility change, and when it happens we would record that in the compatibility data.
> 
> Does this capture the intent?

Yes,
Thanks!
